ios/Podfile

```diff
@@ -28,6 +28,8 @@ target 'JitsiMeet' do
   ]
   pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
 
+  pod 'Amplitude-iOS', '~> 4.0.4'
```

Comment:
Can you move this to right before Dropbox official? The order is: RN dependencies, standalone dependencies, RN plugins.